### PR TITLE
Auto release from master branch via travis

### PR DESCRIPTION
**Pull request by leggetter**

### CODE REVIEW

**Comment by leggetter** [comment]
Attempts to fix #95

**Comment by cbetta** [comment]
@leggetter so I was thinking there's 2 ways to approach this:

1. Close to what you have now: On every push to Master it builds and releases a new NPM release. It should then also probably create a tag for the new version on Github. This is close to what you have now. Outstanding issues for this are: tagging on GH, and how to handle a new submission to master with the same version number.
2. My preference: Instead of listening to master we listen to tags. Tags are only pushed once and should never be updated. When someone creates a tag on GH we have Travis pick it up and do a publish to NPM. Yes, it requires an explicit push, and still requires us to also update the version number in package.json before we tag, but it's a lot more explicit.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+- 5
+- 6
+deploy:
+  provider: npm
+  email: devrel@nexmo.com
+  api_key:
+    secure: dIwjf+unoDvAX/oId0TiXZ1/KNgzxZ3sqk/nquKVu0Ln5hEmePiuFEYUlpI7Lz3xVgwJqvlOJwdqGZjTT1xuih9FtOTjYxmAmF+KvfQxlMmHMpQAiOOQryLbEPMJg4lHoCinqOaw9I/cWt0Dad2+OHvzdJ+hBYC9Q54vYguBj1Dq2ENf47Ay0T2w+s6AFDOEvDF7vprFFJFY9xGhTgvJR2AuixaffKMi06CA3+G7WyNCFyYGfZgMdsx+E6rh2RC6ZntWOxR/+DbNROF96RQ4zyPflPk52A5yGeIal15tfegF4ZZ912escYeEfXn2DM2xe98qcnUBexYRGCvQQYI0E3rUDs+wrAI84NeylZEVnDXRnRgBpx8KxAF3vBsh2v+FeWySGEy+w9stISXNuWx7VkhaKd8GDicATILE8jRMivu+LmObslrLWbFIn50j6yNltWZjcioDHAuUavIsHYAb42dtg0STpnAZXFkxv8lrIoJehGURRRl9sjaokHZXM7BoQNL08Jfyi6DCage9JokpRalkg89tlAkYynuM5dBaqJopQrr3+gpLmxUhYBik1nCR9/TF8NxcaoeRzLlam4anerXEz5MECtymnOKH9nYdiCFCftOWSgVJ3pZYmq1SXmMEAAFEVGHs2K8sA/9gmNFfseHAoQq4vMkR0rYnrCYcCkU=
+  on:
+    tags: true
+    branch: master
```
Comment (cbetta) [inline .travis.yml]: We should probably drop this line and only do this on tag

**Review by cbetta — CHANGES_REQUESTED** [review]

**Comment by leggetter** [comment]
@cbetta concerns addressed in 409648b

**Review by cbetta — APPROVED** [review]

**Comment by cbetta** [comment]
Looks good!

**Comment by cbetta** [comment]
As this works on the CLI I say we can merge this.

**Comment by leggetter** [comment]
Sorry - I forgot I was the one who was supposed to merge this 😊

**Pull request merged**